.travis.yml

```diff
@@ -136,4 +136,11 @@ script:
       sh ./misc/travis/clang-format.sh;
     else
       scons -j2 CC=$CC CXX=$CXX platform=$PLATFORM tools=$TOOLS target=$TARGET $OPTIONS $EXTRA_ARGS;
+
+      if [ "$TEST_PROJECT" = "yes" ]; then
+        mkdir -p "$HOME/.local/share/godot/templates/3.2.dev/";
```

Comment:
It would be better to set the path to a custom export template in the `export_presets.cfg`, so that we don't have to hardcode and bump the engine version all the time.

Reply:
You can set a dummy path in the git repo, and then replace it here with sed.